tools/rts_verifier/operation.py

```diff
@@ -96,7 +96,7 @@ def __extract_pointer(self, prefix, line):
             matches = re.search(f'{prefix}=(0x[A-Za-z0-9]+)', line)
 
         if matches is None:
-            raise Exception("failed to parse address string")
+            raise Exception(f"failed to parse address string with prefix '{prefix}' in '{line}'")
```

Comment:
Could you help me understand this? I'm not familiar with the `f"<string>"` semantic.

Reply:
It is just a way to print variables with Python 3.x, see the documentation [here](https://docs.python.org/3/reference/lexical_analysis.html#f-strings). I believe we are not always consistent in the way we print variables in a string when using Python, unfortunately.

I added the variables to make the debugging experience easier.